resources/lang/id/home.php

```diff
@@ -0,0 +1,152 @@
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'landing' => [
+        'download' => 'Unduh sekarang',
+        'online' => '<strong>:players</strong> saat ini daring dalam <strong>:games</strong> permainan',
+        'peak' => 'Terbanyak, :count pengguna daring',
+        'players' => '<strong>:count</strong> pengguna terdaftar',
+
+        'slogan' => [
+            'main' => 'game ritme free-to-win terbaik',
+            'sub' => 'rhythm is just a click away',
+        ],
+    ],
+
+    'search' => [
+        'advanced_link' => 'Pencarian lanjutan',
+        'button' => 'Cari',
+        'empty_result' => 'Tidak Ditemukan!',
+        'missing_query' => 'Kata kunci pencarian harus mengandung setidaknya :n karakter',
+        'placeholder' => 'ketik untuk mencari',
+        'title' => 'Pencarian',
+
+        'beatmapset' => [
+            'more' => ':count hasil pencarian beatmap lainnya',
+            'more_simple' => 'Lihat hasil pencarian beatmap lainnya',
+            'title' => 'Beatmaps',
+        ],
+
+        'forum_post' => [
+            'all' => 'Semua forum',
+            'link' => 'Cari di forum',
+            'more_simple' => 'Lihat hasil pencarian forum lainnya',
+            'title' => 'Forum',
+
+            'label' => [
+                'forum' => 'cari di forum',
+                'forum_children' => 'termasuk subforum',
+                'topic_id' => 'topik #',
+                'username' => 'pemilik',
+            ],
+        ],
+
+        'mode' => [
+            'all' => 'semua',
+            'beatmapset' => 'beatmap',
+            'forum_post' => 'forum',
+            'user' => 'pemain',
+            'wiki_page' => 'wiki',
+        ],
+
+        'user' => [
+            'more' => ':count hasil pencarian pengguna lainnya',
+            'more_simple' => 'Lihat hasil pencarian pengguna lainnya',
+            'more_hidden' => 'Pencarian pengguna terbatas untuk :max pengguna saja. Perbaiki kata kunci pencarian Anda',
+            'title' => 'Pengguna',
+        ],
+
+        'wiki_page' => [
+            'link' => 'Cari di wiki',
+            'more_simple' => 'Lihat hasil pencarian wiki lainnya',
+            'title' => 'Wiki',
+        ],
+    ],
+
+    'download' => [
+        'tagline' => 'mari persiapkan<br>diri Anda!',
+        'action' => 'Unduh osu!',
+        'os' => [
+            'windows' => 'untuk Windows',
+            'macos' => 'untuk macOS',
+            'linux' => 'untuk Linux',
+        ],
+        'mirror' => 'alternatif',
+        'macos-fallback' => 'pengguna macOS',
+        'steps' => [
+            'register' => [
+                'title' => 'buat akun',
+                'description' => 'ikuti petunjuk saat memulai permainan untuk masuk atau membuat akun baru',
+            ],
+            'download' => [
+                'title' => 'unduh permainannya',
+                'description' => 'klik tombol di atas untuk mengunduh penginstal, lalu jalankan!',
```

Comment:
I'm not sure about the "penginstal" here to be honest - something more straightforward like this would work better I think : 

+                'description' => 'klik tombol di atas untuk mengunduh file installer, lalu jalankan!',